OrbitQt/orbitsamplingreport.h

```diff
@@ -17,7 +19,8 @@ class OrbitSamplingReport : public QWidget {
   explicit OrbitSamplingReport(QWidget* parent = nullptr);
   ~OrbitSamplingReport() override;
 
-  void Initialize(std::shared_ptr<class SamplingReport> a_Report);
+  void Initialize(DataView* callstack_data_view,
+                  std::shared_ptr<class SamplingReport> a_Report);
```

Comment:
As above,  maybe align the two parameters to the same name style?

Reply:
done